hack/hive-dev-install.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+HIVE_OPERATOR_NS="hive"
+KUBECTL=$( which kubectl 2> /dev/null || which oc 2> /dev/null)
+
+function cleanup {
+	[ -f "$(pwd)/kubectl" ] && rm -f "$(pwd)/kubectl"
+}
+
+function download_tmp_kubectl {
+	curl -sLO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+	if [ $? -ne 0 ]; then
+		echo ": error downloading kubectl"
+		exit
+	fi
+	chmod 755 kubectl
+	KUBECTL="$(pwd)/kubectl"
+}
+
+function verify_tools {
+	if [ ! -z "$KUBECTL" ]; then
+		return
+	fi
+	echo -n "kubectl or oc not detected, downloading"
+	download_tmp_kubectl
+	echo ", done."
+
+	if [ $( $KUBECTL get nodes 2>/dev/null | wc -l ) -eq 0 ]; then
+		echo "unable to connect to the cluster"
+		exit
+	fi
+}
+
+set -e
+trap cleanup EXIT
+
+if [ ! -f go.mod ] || [ ! -d ".git" ]; then
+	echo "this script must by run from the repo's root directory"
+	exit 1
+fi
```

Comment:
Would it make sense to extract this block code to a function in another script, as it is used in two different places (here and in hack/hive-generate-config.sh)? 
Also, minor difference about the messages:

- here:
```bash
echo "this script must by run from the repo's root directory"
```

- hack/hive-generate-config.sh:
```bash
echo "this script must by run from the repo root directory"
```

Reply:
Fixed the typo however, with regards to the duplicated directory check, I think having a separate file with a single 4 line function in it sacrifices readability for complexity in this instance.